package-lock.json

```diff
@@ -18903,6 +18896,25 @@
                 "assert": "^2.1.0"
             }
         },
+        "packages/q": {
+            "name": "amazonq",
+            "version": "1.0.0-SNAPSHOT",
+            "extraneous": true,
+            "license": "Apache-2.0",
+            "devDependencies": {},
+            "engines": {
+                "npm": "^10.1.0",
+                "vscode": "^1.68.0"
+            }
+        },
+        "packages/toolkit": {
+            "name": "aws-toolkit-vscode",
+            "version": "2.12.0-SNAPSHOT",
+            "license": "Apache-2.0",
+            "devDependencies": {
+                "aws-core-vscode": "file:../core/"
```

Comment:
Should it be in regular "dependencies" instead of "devDependencies" ?

Reply:
yes 👍